bcipy/gui/mode/RSVPKeyboard.py

```diff
@@ -45,8 +50,8 @@ def launch_bci_main(self, event: wx.Event) -> None:
             username = self.comboboxes[0].GetValue().replace(" ", "_")
             experiment_type = event.GetEventObject().GetId()
             mode = 'RSVP'
-            cmd = 'python bci_main.py -m {} -t {} -u {}'.format(
-                mode, experiment_type, username)
+            cmd = 'python bci_main.py -m {} -t {} -u {} -p {}'.format(
+                mode, experiment_type, username, self.PARAMETER_LOCATION)
```

Comment:
Now that we're using python 3 we can use f-strings.

    cmd = f'python bci_main.py -m {mode} -t {experiment_type} -u {username} -p {self.parameter_location}'